Resources/doc/reference/advanced_configuration.rst

```diff
@@ -120,6 +120,35 @@ application's config file:
                     model_manager:          # dependency name, from the table above
                         sonata.order.admin.order.manager  # customised service id
 
+
+Using Annotations to define Admins
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
it wasn't a big change: https://gist.github.com/wodka/95457be1ccaea2d0efd8

-> Sonata\AdminBundle\Tests\Command\GenerateAdminCommandTest is failing now all the time because container is never set


easy fix is to extend the setup: 
```php
        $kernel->expects($this->any())
            ->method('getContainer')
            ->will($this->returnValue($this->container));
```